File: core-java-practice/leet-code-codebase/areatriangle.java
import java.util.*;
public class areatriangle {
    public static double calculateArea(double base, double height) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter base: ");
        base = sc.nextDouble();
        System.out.print("Enter height: ");
        height = sc.nextDouble();
        sc.close();
        return (base * height) / 2;
    }

    public static void main(String[] args) {
        double base = 10;
        double height = 5;
        double area = calculateArea(base, height);
        System.out.println("Area of triangle: " + area);
    }
}

File: core-csharp-practice/gcr-codebase/csharp-programming-elements/level1/AreaTriangle.cs
using System;

class AreaTriangle
{
    static void Main(string[] args)
    {
        
        Console.WriteLine("Enter the base of the triangle (in inches):");
        double base_val = double.Parse(Console.ReadLine());

        Console.WriteLine("Enter the height of the triangle (in inches):");
        double height = double.Parse(Console.ReadLine());

        double areaSquareInches = 0.5 * base_val * height;
        double areaSquareCm = areaSquareInches * 6.4516;

        Console.WriteLine($"Area of triangle in square inches: {areaSquareInches}");
        Console.WriteLine($"Area of triangle in square centimeters: {areaSquareCm}");
    }   
}